wandb_logger.py

```diff
@@ -19,6 +23,92 @@
 wandb.init(project='catlearn', config={})
 
 
+def compute_kge_metrics(
```

Comment:
This is more or less what I planned to do (separate metrics.py file). I didn't for now since the PR as it is is not supposed to be merged yet. But duly noted.